packages/core/htpasswd/src/htpasswd.ts

```diff
@@ -11,12 +11,18 @@ import {
   addUserToHTPasswd,
   changePasswordToHTPasswd,
   sanityCheck,
+  HtpasswdHashAlgorithm,
+  HtpasswdHashConfig,
 } from './utils';
 
 export type HTPasswdConfig = {
   file: string;
+  algorithm?: string;
```

Comment:
Maybe if we use an enum? or an union type? `'crypt|sha1|md5|bcrypt` , just in order provide more context.

Reply:
Oh, we have the type just below.

Reply:
Sure, fixed!